src/main/java/core/basesyntax/AreaCalculable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculable {
```

Comment:
Consider renaming the interface to better reflect its purpose without using a verb. For example, 'AreaCalculator' would be a more appropriate name than 'AreaCalculable'.